src/managing-workflow/tuning-component-settings.md

```diff
@@ -0,0 +1,228 @@
+SYSLOG_PORT_1     | The port of a remote syslog endpoint for shipping logs
+
+## Customizing the Logger
+
+The following environment variables are tunable for the [Logger][] component:
+
+Setting           | Description
+----------------- | ---------------------------------
+STORAGE_ADAPTER   | How to store logs that are sent to the logger (default: "memory")
+NUMBER_OF_LINES   | How many lines to store in the ring buffer (default: 1000)
+
+## Customizing Minio
+
+The following environment variables are tunable for [Minio][]:
+
+Setting           | Description
+----------------- | ---------------------------------
+
+## Customizing the Monitor
+
+The following environment variables are tunable for the [Monitor][] component. Please see
+[telegraf configuration](https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md)
+for more information on these settings.
+
+	AGENT_INTERVAL
+	AGENT_ROUND_INTERVAL
+	AGENT_BUFFER_LIMIT
+	AGENT_COLLECTION_JITTER
+	AGENT_FLUSH_INTERVAL
+	AGENT_FLUSH_JITTER
+	AGENT_DEBUG
+	AGENT_QUIET
+	AGENT_FLUSH_BUFFER
+	AGENT_HOSTNAME
+	AMON_SERVER_KEY
+	AMON_INSTANCE
+	AMON_TIMEOUT
+	AMQP_URL
+	AMQP_EXCHANGE
+	AMQP_ROUTING_TAG
+	AMQP_SSL_CA
+	AMQP_SSL_CERT
+	AMQP_SSL_KEY
+	AMQP_RETENTION_POLICY
+	AMQP_DATABASE
+	AMQP_PRECISION
+	DATADOG_API_KEY
+	GRAPHITE_SERVERS
+	GRAPHITE_PREFIX
+	GRAPHITE_TIMEOUT
+	INFLUXDB_URLS
+	INFLUXDB_DATABASE
+	INFLUXDB_PRECISION
+	INFLUXDB_TIMEOUT
+	INFLUXDB_USERNAME
+	INFLUXDB_PASSWORD
+	INFLUXDB_USER_AGENT
+	INFLUXDB_UDP_PAYLOAD
+	KAFKA_BROKERS
+	KAFKA_TOPIC
+	KAFKA_ROUTING_TAG
+	KAFKA_CERTIFICATE
+	KAFKA_KEY
+	KAFKA_CA
+	KAFKA_VERIFY_SSL
+	LIBRATO_API_USER
+	LIBRATO_API_TOKEN
+	LIBRATO_SOURCE_TAG
+	NSQ_SERVER
+	NSQ_TOPIC
+	OPEN_TSDB_PREFIX
+	OPEN_TSDB_HOST
+	OPEN_TSDB_PORT
+	OPEN_TSDB_DEBUG
+	RIEMANN_URL
+	RIEMANN_TRANSPORT
+	ENABLE_ZFS
+	DISK_MOUNT_POINTS
+	DISKIO_DEVICES
+	DISKIO_SKIP_SERIAL_NUMBER
+	NET_INTERFACES
+	APACHE_URLS
+	DISQUE_SERVERS
+	DOCKER_ENDPOINT
+	DOCKER_CONTAINER_NAMES
+	ELASTIC_SEARCH_SERVERS
+	ELASTIC_SEARCH_LOCAL
+	ELASTIC_SEARCH_CLUSTER_HEALTH
+	HAPROXY_SERVERS
+	HTTP_JSON_NAME
+	HTTP_JSON_SERVERS
+	HTTP_JSON_METHOD
+	HTTP_JSON_TAG_KEYS
+	HTTP_JSON_EVENT_TYPE
+	HTTP_JSON_THRESHOLD
+	ENABLE_INFLUXDB_INPUT
+	INFLUXDB_URLS
+	MEMCACHED_SERVERS
+	MONGODB_SERVERS
+	MYSQL_SERVERS
+	NGINX_URLS
+	NSQ_ENDPOINTS
+	POSTGRESQL_ADDRESS
+	POSTGRESQL_DATABASES
+	PROMETHEUS_URLS
+	PROMETHEUS_INSECURE_SKIP_VERIFY
+	PROMETHEUS_BEARER_TOKEN
+	RABBITMQ_URL
+	RABBITMQ_NAME
+	RABBITMQ_USERNAME
+	RABBITMQ_PASSWORD
+	RABBITMQ_NODES
+	REDIS_SERVERS
+	RETHINKDB_SERVERS
+	ZOOKEEPER_SERVERS
+	KAFKA_CONSUMER_TOPICS
+	KAFKA_CONSUMER_ZOOKEEPER_PEERS
+	KAFKA_CONSUMER_GROUP
+	KAFKA_CONSUMER_POINT_BUFFER
+	KAFKA_CONSUMER_OFFSET
+	STATSD_SERVICE_ADDRESS
+	STATSD_DELETE_GAUGES
+	STATSD_DELETE_COUNTERS
+	STATSD_DELETE_SETS
+	STATSD_DELETE_TIMINGS
+	STATSD_PERCENTILES
+	STATSD_CONVERT_NAMES
+	STATSD_TEMPLATES
+	STATSD_ALLOWED_PENDING_MESSAGES
+	STATSD_PERCENTILE_LIMIT
+	STATSD_UDP_PACKET_SIZE
```

Comment:
wowza. is it worth listing all of these rather than _just_ deferring to the telegraf doc? it seems odd to tell people that the vars exist, but then direct them to a doc to discover what they mean.